lib/rubocop/cop/style/method_call_with_args_parentheses/omit_parentheses.rb

```diff
@@ -177,6 +180,18 @@ def hash_literal_in_arguments?(node)
             end
           end
 
+          def ambiguous_range_argument?(node)
+            if (first_arg = node.first_argument)&.range_type? && first_arg.begin.nil?
+              return true
+            end
+
+            if (last_arg = node.last_argument)&.range_type? && last_arg.end.nil?
+              return true
+            end
```

Comment:
That's just my two cents. Could you update it to a style that reflects guard conditions?

```suggestion
            return true if (first_arg = node.first_argument)&.range_type? && first_arg.begin.nil?
            return true if (last_arg = node.last_argument)&.range_type? && last_arg.end.nil?
```